addons/mail/static/tests/helpers/mock_models.js

```diff
@@ -124,7 +124,7 @@ class MockModels {
                     attachment_ids: { string: "Attachments", type: 'many2many', relation: 'ir.attachment', default: [] },
                     author_id: { string: "Author", type: 'many2one', relation: 'res.partner', default() { return this.currentPartnerId; } },
                     body: { string: "Contents", type: 'html', default: "<p></p>" },
-                    date: { string: "Date", type: 'datetime' },
+                    date: { string: "Date", type: 'datetime', default: moment.utc().format("YYYY-MM-DD HH:mm:ss") },
```

Comment:
This will use the same date for all tests, the one when this file was read.

Instead it should be a function returning the date so that it's actually dynamic.